test/integration/integration.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const find = require('find');
+const utils = require('./utils');
+const setup = require('./setup');
+
+const totalPeers = Number.parseInt(process.env.TOTAL_PEERS) || 10;
+// Each peer connected to 9 other pairs and have 2 connection for bi-directional communication
```

Comment:
Not clear

Reply:
@jondubois how can we improve this explanation?